src/test/java/calculator/StringCalculatorTest.java

```diff
@@ -0,0 +1,20 @@
+package calculator;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class StringCalculatorTest {
+
+    private StringCalculator stringCalculator;
```

Comment:
상태를 가지지 않는 클래스는 인스턴스 변수로 선언할 필요가 없을것 같아요.

Reply:
`StringCalculator` 가 상태를 가지게 되어 객체 생성하는 코드가 필요할것 같아요.
@BeforeEach 와 같은 SetUp 메소드를 활용하여 테스트 코드에서 필요한 객체를 생성해보세요 !
```suggestion
    private StringCalculator stringCalculator;
```

```
@BeforeEach
    void setUp() {
        stringCalculator = new StringCalculator();
    }
```